scripts/asm-installer/install_asm

```diff
@@ -1411,6 +1444,23 @@ is_workload_identity_enabled() {
   if [[ "${ENABLED}" = 'null' ]]; then false; fi
 }
 
+get_hub_workload_identity() {
+  local ID_NUMBER
+  ID_NUMBER=`kubectl get membership -o=json | jq '.items | length'`
+  if [[ ${ID_NUMBER} -gt 1 ]]; then
+    fatal "Found more than one Hub Membership. Please make sure one cluster is associated with one Hub Membership.";
```

Comment:
Is this possible?

Reply:
I discussed this topic with GKE HUB team. `more than one Hub Membership`, if customer really wants, there are ways around it.  But it is not standard supported ways to register the cluster as there is exclusivity check enforced on client side. And I changed the command as they suggested. We do not need to worry about multiple Memberships case.